en/docs/administer/blue-green-data-encryption-keyrotation.md

```diff
@@ -0,0 +1,252 @@
+    *   newRegUsername - `<NEW_IS_HOME>` registry database username
+    *   newRegPassword - `<NEW_IS_HOME>` registry database password 
+    !!! note
+        Encode the `<NEW_IS_HOME>` registry database plaintext password in [base64](https://www.base64encode.org/) and insert here
+    *   enableDBMigrator - Enable/disable re-encryption for the identity and registry databases
+    !!! note
+        Keep this to always **true** to avoid unnecessary issues
+    *   enableConfigMigrator - Enable/disable re-encryption for the configuration files
+    !!! note
+        Keep this to always **true** to avoid unnecessary issues
+    *   enableSyncMigrator - Enable/disable syncing mechanism
+    !!! note
+        You only need to set this to **true**, if you have opted in for blue-green key rotation with zero downtime for the end user flows
+
+    ??? tip "Sample configuration written for the properties.yaml file"
+            
+        ```
+        oldSecretKey: AFA27B44D43B02A9FEA41D13CEDC2E40
+        newSecretKey: 1fc0bc7a3805b42afa5f5af07a595f56
+        newISHome: /home/geesa/Desktop/IS/copy/mysql/wso2is-5.11.0
+        oldIdnDBUrl: jdbc:mysql://localhost:3306/regdb?useSSL=false
+        oldIdnUsername: root
+        oldIdnPassword: cm9vdA==
+        newIdnDBUrl: jdbc:mysql://localhost:3306/regdb1?useSSL=false
+        newIdnUsername: root
+        newIdnPassword: cm9vdA==
+        newRegDBUrl: jdbc:mysql://localhost:3306/regdb1?useSSL=false
+        newRegUsername: root
+        newRegPassword: cm9vdA==
+        enableDBMigrator: true
+        enableConfigMigrator: true
+        enableSyncMigrator: true
+        ```
+    ??? tip "Sample configuration written for H2 DB type URL"
+            
+        ```
+        jdbc:h2:~/Desktop/IS/copy/wso2is-5.11.0-beta5-SNAPSHOT/repository/database/WSO2IDENTITY_DB
+        ```
+    ??? tip "Sample configuration written for MySQL DB type URL"
+        
+        ```
+        jdbc:mysql://localhost:3306/idndb?useSSL=false
+        ```
+    ??? tip "Sample configuration written for DB2 DB type URL"
+        
+        ```
+        jdbc:db2://localhost:50000/idndb
+        ```
+    ??? tip "Sample configuration written for PostgreSQL DB type URL"
+        
+        ```
+        jdbc:postgresql://localhost:5432/idndb
+        ```
+    ??? tip "Sample configuration written for MSSQL DB type URL"
+        
+        ```
+        jdbc:sqlserver://localhost:1433;databaseName=idndb
+        ```
+    ??? tip "Sample configuration written for Oracle DB type URL"
+        
+        ```
+        jdbc:oracle:thin:@localhost:1521/ORCLCDB.LOCALDOMAIN
+        ```
+
+6.  Run the tool using `./keyrotation.sh keyrotation-tool-<version>-SNAPSHOT.jar properties.yaml` command from  inside the `<KEY_ROTATION_TOOL>` location
+
+7.  Edit the `<NEW_IS_HOME>` `deployment.toml` file having the new configured databases and the new key 
+
+8.  Start the `<NEW_IS_HOME>` once existing DB and config file data re-encryption completes
```

Comment:
```suggestion
8.  Start `<NEW_IS_HOME>` once existing DB and config file data re-encryption is done. 
```

Reply:
Addressed with https://github.com/wso2/docs-is/pull/2555/commits/e4826286f4b66bb74b49890f19dce24148b95808